applications/nrf5340_audio/doc/requirements.rst

```diff
@@ -9,17 +9,15 @@ nRF5340 Audio application requirements
 
 The nRF5340 Audio applications are designed to be used only with the following hardware:
 
-.. table-from-rows:: /includes/sample_board_rows.txt
-   :header: heading
-   :rows: nrf5340_audio_dk_nrf5340
+.. ../table-from-sample-yaml::
```

Comment:
This won't work because of `../`. The tag is supposed to look at the sample.yaml at the same dir level. This is not the case here, so we can't use this formatting.

```suggestion
.. table-from-rows:: /includes/sample_board_rows.txt
   :header: heading
   :rows: nrf5340_audio_dk_nrf5340, nrf5340dk_nrf5340_cpuapp
```